clients/python/provider/superposition_provider/configuration_client.py

```
@@ -218,7 +218,7 @@ def _to_float(self, value: Any, default: float) -> float:
         except (TypeError, ValueError):
             return default
     def _to_object(self, value: Any, default: object) -> object:
-        if isinstance(value, dict): return value
+        if isinstance(value, object): return value
```

Ruff (0.15.0)

[error] Multiple statements on one line (colon)

(E701)